tasks/main.yml

```diff
@@ -15,3 +22,19 @@
     owner: root
     group: root
     mode: 0600
+  when: ansible_pkg_mgr == "apt"
+
+- name: Copy Redis conf 
+  template:
+    src: redis.conf.j2
+    dest: /etc/redis.conf
+    owner: root
+    group: root
+    mode: 0600
+  when: ansible_pkg_mgr == "pacman"
+
+- name: Restart redis service
```

Comment:
use a handler for this

Reply:
no there is no advantage to add a handler since its only called one time and also enables the service

Reply:
With the current implementation, the service will be restarted each time the playbook runs, even if there is no need to.
The handler would run every time the config changes, including the first time this role is executed.